common/stacks/src/test/scala/weco/api/stacks/services/WorkLookupTest.scala

```diff
@@ -18,38 +18,38 @@ class WorkLookupTest
     with WorkGenerators {
   val lookup: WorkLookup = WorkLookup(elasticClient)
 
-  it("returns a work with matching ID") {
-    withLocalWorksIndex { index =>
-      val work = indexedWork()
-
-      insertIntoElasticsearch(index, work)
-
-      val future = lookup.byCanonicalId(work.state.canonicalId)(index)
-
-      whenReady(future) {
-        _ shouldBe Right(work)
-      }
-    }
-  }
-
-  it("returns None if there is no such work") {
-    withLocalWorksIndex { index =>
-      val id = createCanonicalId
-      val future = lookup.byCanonicalId(id)(index)
-
-      whenReady(future) {
-        _ shouldBe Left(DocumentNotFoundError(id))
-      }
-    }
-  }
-
-  it("returns Left[ElasticError] if Elasticsearch has an error") {
-    val index = createIndex
-    val future = lookup.byCanonicalId(createCanonicalId)(index)
-
-    whenReady(future) { err =>
-      err.left.value shouldBe a[IndexNotFoundError]
-      err.left.value.asInstanceOf[IndexNotFoundError].index shouldBe index.name
-    }
-  }
+//  it("returns a work with matching ID") {
```

Comment:
Should these be removed?

Reply:
Ah not, they should not have been commented!